apps/scan/backend/src/scanners/custom/app_config.test.ts

```diff
@@ -295,7 +295,7 @@ test('export CVRs to USB in deprecated VotingWorks format', async () => {
   );
 });
 
-test('setPrecinctSelection will reset polls to closed and update auth instance', async () => {
```

Comment:
A relic of a much earlier iteration of auth code

Reply:
![Untitled](https://user-images.githubusercontent.com/1938/231017131-845b457c-7906-4712-a5dc-f22508965f2d.jpg)